### PR TITLE
Optimize deploy multiple operator

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added, or deleted? (Required)
Optimize deploy multiple operator

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/deploy-multiple-tidb-operator.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 summary: 介绍如何部署多套 TiDB Operator 分别管理不同的 TiDB 集群。
 aliases: ['/zh/tidb-in-kubernetes/dev/canary-deployment-tidb-operator/']
 ---
 
-# 部署多套 TiDB Operator 分别管理不同的 TiDB 集群
+# 使用多套 TiDB Operator 单独管理不同的 TiDB 集群
+
+你可使用一套 TiDB Operator 管理多个 TiDB 集群。如果你有以下业务需求，可以部署多套 TiDB Operator 分别管理不同的 TiDB 集群：
+
+- 业务中有多个版本的 TiDB 集群，需要单独管理不同版本的集群
```
Comment (csuzhangxc) [inline zh/deploy-multiple-tidb-operator.md:11]: 一套 TiDB Operator 也可以管理多个不同版本的 TiDB 集群

**Review by csuzhangxc — APPROVED** [review]

File: zh/deploy-multiple-tidb-operator.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 summary: 介绍如何部署多套 TiDB Operator 分别管理不同的 TiDB 集群。
 aliases: ['/zh/tidb-in-kubernetes/dev/canary-deployment-tidb-operator/']
 ---
 
-# 部署多套 TiDB Operator 分别管理不同的 TiDB 集群
+# 使用多套 TiDB Operator 单独管理不同的 TiDB 集群
+
+你可使用一套 TiDB Operator 管理多个 TiDB 集群。如果你有以下业务需求，可以部署多套 TiDB Operator 分别管理不同的 TiDB 集群：
+
+- 需要灰度升级 TiDB Operator 至新版本，避免新版本存在潜在问题影响业务
```
Comment (ran-huang) [inline zh/deploy-multiple-tidb-operator.md:11]:
```suggestion
- 需要[灰度升级 TiDB Operator](canary-upgrade-tidb-operator.md) 至新版本，避免新版本存在潜在问题影响业务
```

File: zh/deploy-multiple-tidb-operator.md [hunk 3]
```diff
@@ -181,3 +149,42 @@ aliases: ['/zh/tidb-in-kubernetes/dev/canary-deployment-tidb-operator/']
...
+
+如果需要部署第三套或更多 TiDB Operator 来管理集群，重复第 3 步、第 4 步和第 5 步的操作即可。
+
+## 相关参数
+
+为了支持部署多套 TiDB Operator，`tidb-operator` chart 中 `values.yaml` 文件里面添加了以下参数。
+
+- `appendReleaseSuffix`
+
+    如果配置为 `true`，部署时会自动为 `tidb-controller-manager` 和 `tidb-scheduler` 相关的资源名称添加后缀 `-{{ .Release.Name }}`，例如，通过 `helm install canary pingcap/tidb-operator ...` 命令部署的 `tidb-controller-manager` deployment 名称为：`tidb-controller-manager-canary`，如果要部署多套 TiDB Operator 需要开启此参数。
```
Comment (ran-huang) [inline zh/deploy-multiple-tidb-operator.md:161]:
```suggestion
    如果配置为 `true`，部署时会自动为 `tidb-controller-manager` 和 `tidb-scheduler` 相关的资源名称添加后缀 `-{{ .Release.Name }}`。例如，通过 `helm install canary pingcap/tidb-operator ...` 命令部署的 `tidb-controller-manager` deployment 名称为：`tidb-controller-manager-canary`。如果要部署多套 TiDB Operator，需要开启此参数。
```

File: zh/deploy-multiple-tidb-operator.md [hunk 4]
```diff
@@ -181,3 +149,42 @@ aliases: ['/zh/tidb-in-kubernetes/dev/canary-deployment-tidb-operator/']
...
+    ```yaml
+    selector:
+    - canary-release=v1
+    - k1==v1
+    - k2!=v2
+    ```
+
+- `scheduler.create`
+
+    控制是否创建 `tidb-scheduler`。
```
Comment (ran-huang) [inline zh/deploy-multiple-tidb-operator.md:188]:
```suggestion
    控制是否创建 `tidb-scheduler`。在使用 TiDB Operator 时，`tidb-scheduler` 并不是必须使用。你可以参考 [tidb-scheduler 与 default-scheduler](tidb-scheduler.md#tidb-scheduler-与-default-scheduler)，确认是否需要部署 `tidb-scheduler`。
```

**Review by ran-huang — APPROVED** [review]
LGTM

File: en/deploy-multiple-tidb-operator.md [hunk 5]
```diff
@@ -184,3 +148,46 @@ To support deploying multiple sets of TiDB Operator, the following parameters ar
     </details>
 
     By comparing the logs of the two sets of TiDB Operator, you can confirm that the first TiDB Operator only manages the `tidb-cluster-1/basic1` cluster, and the second TiDB Operator only manages the `tidb-cluster-2/basic2` cluster.
+
+If you want to deploy a third or more sets of TiDB Operator, repeat step 3, step 4, and step 5.
+
+## Related parameters
+
+To support deploying multiple sets of TiDB Operator, the following parameters are added to the `values.yaml` file in the `tidb-operator` chart:
```
Comment (ran-huang) [inline en/deploy-multiple-tidb-operator.md:156]:
```suggestion
In the `values.yaml` file in the `tidb-operator` chart, the following parameters are related to the deployment of multiple sets of TiDB Operator:
```

File: zh/deploy-multiple-tidb-operator.md [hunk 6]
```diff
@@ -181,3 +149,42 @@ aliases: ['/zh/tidb-in-kubernetes/dev/canary-deployment-tidb-operator/']
     </details>
 
     通过对比两套 TiDB Operator tidb-controller-manager 日志，第一套 TiDB Operator 仅管理 `tidb-cluster-1/basic1` 集群，第二套 TiDB Operator 仅管理 `tidb-cluster-2/basic2` 集群。
+
+如果需要部署第三套或更多 TiDB Operator 来管理集群，重复第 3 步、第 4 步和第 5 步的操作即可。
+
+## 相关参数
+
+为了支持部署多套 TiDB Operator，`tidb-operator` chart 中 `values.yaml` 文件里面添加了以下参数。
```
Comment (ran-huang) [inline zh/deploy-multiple-tidb-operator.md:157]:
```suggestion
`tidb-operator` chart 的 `values.yaml` 文件里，有以下参数与部署多套 TiDB Operator 相关。
```

Comment (ran-huang) [reply zh/deploy-multiple-tidb-operator.md:157]: 感觉这样更贴近用户的视角。“为了支持......添加了以下参数“对用户是一个无关信息，增加负担。

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**